flexget/plugins/filter/series.py

```diff
@@ -483,7 +538,7 @@ def get_latest_release(series, downloaded=True, season=None):
     :param Season: season to find newest release for
     :return: Instance of Episode or None if not found.
     """
-    session = Session.object_session(series)
```

Comment:
An oversight. I had some issues with session due to some errors I made and I tested some changes. Reverted back.